### PR TITLE
reuse compliance

### PR DESCRIPTION
**Pull request by TorstenD-SAP**
**Description**

Changes proposed in this pull request:

- Correct .reuse/deb5
- Add LICENSES folder and Apache 2.0 license file
- Add reuse badge to the README

**Related issue(s)**

### CODE REVIEW

File: .reuse/dep5 [hunk 1]
```diff
@@ -1,11 +1,11 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
-Upstream-Name: <YOUR-REPO-NAME>
-Upstream-Contact: <YOUR-CONTACT (MAIL ADDRESS ETC.)>
-Source: <https://github.com/sap/YOUR-REPO-NAME>
-Disclaimer: The code in this project may include calls to APIs ("API Calls") of
+Upstream-Name: kyma-environment-broker
```
Comment (Disper) [inline .reuse/dep5:2]: this probably should be a `cloud-manager` instead

**Review by ngrkajac — COMMENTED** [review]
for each kyma-environment-broker, it should go cloud-manager, I suppose?

File: .reuse/dep5 [hunk 2]
```diff
@@ -1,11 +1,11 @@
...
+Upstream-Contact: team-gopher@sap.com
```
Comment (ngrkajac) [inline .reuse/dep5]: should we add Phoenix DL list here?

DL Phoenix Team: DL_623B716E6A0D2D029907BEDC@global.corp.sap

File: .reuse/dep5 (hunk 1 above)
Comment (TorstenD-SAP) [reply .reuse/dep5]: Changed.

**Review by ngrkajac — APPROVED** [review]

**Pull request merged**